integration-tests/integration_tests/integration_tests/component_tests/component_asynchronous_reliable_message_pattern_tests.py

```diff
@@ -110,7 +155,7 @@ def test_should_record_message_status_as_nackd_when_ebxml_error_response_returne
                 'WORKFLOW': 'async-reliable'
             })
 
-    def test_should_return_information_from_a_soap_fault_returned_from_spine_in_original_post_request__to_client_when_sync_async_requested(self):
+    def test_should_return_information_from_a_soap_fault_returned_from_spine_in_original_post_request_to_client_when_sync_async_requested(self):
```

Comment:
I was going to correct this in mine but it might be worth doing it as a part of this to get it right earlier, but this test should be:
`test_should_return_information_from_a_soap_fault_returned_from_spine_in_original_post_request_to_client_when_sync_async_requested`